### PR TITLE
feat(android): document how to use renderdoc with Android

### PR DESCRIPTION
**Pull request by morphis**

### CODE REVIEW

**Comment by keirthana** [comment]
You can ignore the spellcheck error, I have added it to the ignore-list as part of the release notes PR.

File: howto/android/graphics-debugging-with-renderdoc.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+[Renderdoc](https://github.com/baldurk/renderdoc) is a graphics frame debugger which supports Android applications.
+
+Debugging graphics can be challenging and [Renderdoc](https://github.com/baldurk/renderdoc) is a great tool to provide insight into Vulkan and OpenGL command execution. The following describes how you can use Renderdoc with Anbox Cloud.
```
Comment (ajanon) [inline howto/android/graphics-debugging-with-renderdoc.md:3]: Isn't the first sentence redundant with the second one?

File: howto/android/graphics-debugging-with-renderdoc.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Install Renderdoc
+
+Renderdoc provides binary builds [here](https://renderdoc.org/builds). These come with all prerequisites to be able to capture on Android.
+
+For capturing Android applications you also need the Android SDK as Renderdoc needs a few tools like adb or aapt. The easiest is to install [Android Studio](https://developer.android.com/studio) which will allow you to install the SDK.
```
Comment (ajanon) [inline howto/android/graphics-debugging-with-renderdoc.md:9]: Shouldn't we instead tell people to install from the snap here? We could link to the snapstore https://snapcraft.io/android-studio, or use the CLI to install:
```sh
sudo snap install android-studio --classic
```

What do you think?

File: howto/android/graphics-debugging-with-renderdoc.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Afterwards you connect to Android by running
+
+    adb connect <host IP>:9001
+
+The device will show up in the output of
+
+    adb devices
+
+Renderdoc can only inject itself into applications which have debug mode turn on. See [here](https://renderdoc.org/docs/how/how_android_capture.html#how-do-i-use-renderdoc-on-android) for more details. To be able to debug any application regardless of it being build with debug settings Renderdoc needs to be able to run as root over ADB. For this the `android.enable_adb_root` feature flag can be set to allow this. Afterwards can run Renderdoc with any application.
```
Comment (ajanon) [inline howto/android/graphics-debugging-with-renderdoc.md:29]:
```suggestion
Renderdoc can only inject itself into applications which have debug mode turn on. See [here](https://renderdoc.org/docs/how/how_android_capture.html#how-do-i-use-renderdoc-on-android) for more details. To be able to debug any application regardless of it being build with debug settings Renderdoc needs to be able to run as root over ADB. For this the `android.enable_adb_root` feature flag can be set to allow this. Afterwards you can run Renderdoc with any application.
```

Should we link to the feature flag section of our docs here?

**Review by ajanon — CHANGES_REQUESTED** [review]
Some questions.

File: howto/android/graphics-debugging-with-renderdoc.md (hunk 2 above)
Comment (morphis) [reply howto/android/graphics-debugging-with-renderdoc.md:9]: No, we should only ever point to the official installation instructions of external tools we point to and do not reference custom packaged, not official ways of installing.

File: howto/android/graphics-debugging-with-renderdoc.md (hunk 1 above)
Comment (morphis) [reply howto/android/graphics-debugging-with-renderdoc.md:3]: Yes, I will drop the first one.

**Comment by morphis** [comment]
Refined the document in different ways and it's needed to have another look.

File: howto/android/graphics-debugging-with-renderdoc.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+Debugging graphics can be challenging and [Renderdoc](https://github.com/baldurk/renderdoc) is a great tool to provide insight into Vulkan and OpenGL command execution. The following describes how you can use Renderdoc with Anbox Cloud.
+
+## Install Renderdoc
+
+Renderdoc provides binary builds [here](https://renderdoc.org/builds). These come with all prerequisites to be able to capture on Android.
+
+[note type="caution" status="Warning"]
+The offical Renderdoc builds only support Arm and not x86 Android builds. If you need x86 support you will have to build Renderdoc yourself from source. Please have a look at the official [repository](https://github.com/baldurk/renderdoc) for more information on how to do that.
```
Comment (keirthana) [inline howto/android/graphics-debugging-with-renderdoc.md:8]:
```suggestion
The offical Renderdoc builds only support ARM and not x86 Android builds. If you need x86 support, you will have to build Renderdoc yourself from source. The official [repository](https://github.com/baldurk/renderdoc) for Renderdoc provides more information on how to do that.
```

**Review by keirthana — APPROVED** [review]
Some minor suggestions but otherwise LGTM. So I am approving this.

File: howto/android/graphics-debugging-with-renderdoc.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+[Renderdoc](https://github.com/baldurk/renderdoc) is a graphics frame debugger which supports Android applications.
+
+Debugging graphics can be challenging and [Renderdoc](https://github.com/baldurk/renderdoc) is a great tool to provide insight into Vulkan and OpenGL command execution. The following describes how you can use Renderdoc with Anbox Cloud.
+
+## Install Renderdoc
+
+Renderdoc provides binary builds [here](https://renderdoc.org/builds). These come with all prerequisites to be able to capture on Android.
```
[the remark this answers is not in the document]
Comment (morphis) [reply howto/android/graphics-debugging-with-renderdoc.md:5]: I can rework that, thanks!

**Pull request merged**